doc/guides/backporting-to-release-lines.md

```diff
@@ -8,56 +8,54 @@ pad while preparing a release. The branch name is formatted as follows:
 
 ### Active staging branches
 
-| Release Line | Staging Branch |
-| ------------ | -------------- |
-| `v7.x`       | `v7.x-staging` |
-| `v6.x`       | `v6.x-staging` |
-| `v4.x`       | `v4.x-staging` |
+| Release Line | Staging Branch | LTS Status  |
+| ------------ | -------------- | ----------  |
+| `v8.x`       | `v8.x-staging` | Current     |
+| `v7.x`       | `v7.x-staging` | Maintenance |
+| `v6.x`       | `v6.x-staging` | LTS         |
+| `v4.x`       | `v4.x-staging` | Maintenance |
 
 ## What needs to be backported?
 
 If a cherry-pick from master does not land cleanly on a staging branch, the
 releaser will mark the pull request with a particular label for that release
-line, specifying to our tooling that this pull request should not be included.
-The releaser will then add a comment that a backport is needed.
+line (e.g. `backport-requested-v8.x`), specifying to our tooling that this
+pull request should not be included. The releaser will then add a comment
+that a backport is needed.
 
 ## What can be backported?
 
 The "Current" release line is much more lenient than the LTS release lines in
-what can be landed. Our LTS release lines (v4.x and v6.x as of March 2017)
-require that commits mature in a Current release for at least 2 weeks before
-they can be landed on staging. If the commit can not be cherry-picked from
-master a manual backport will need to be submitted. Please see the [LTS Plan][]
-for more information. After that time, if the commit can be cherry-picked
-cleanly from master, then nothing needs to be done. If not, a backport pull
-request will need to be submitted.
+what can be landed. Our LTS release lines (`v4.x` and `v6.x` as of March 2017)
```

Comment:
How about:

`(`v4.x` and `v6.x` as of March 2017)` -> `(see the [LTS Plan][])` and then remove `Please see the [LTS Plan][] for more information.` from the next line.